tests/spicy/types/bytes/parse-until-eod.spicy

```diff
@@ -0,0 +1,16 @@
+# @TEST-DOC: Validates that `&until` or `&until-including` can be combined with `&eod`.
+#
+# @TEST-EXEC: spicyc -dj %INPUT -o test.hlto
+# @TEST-EXEC: ${SCRIPTS}/printf '0' | spicy-dump -d test.hlto -p Test::X >output
+# @TEST-EXEC: ${SCRIPTS}/printf '0' | spicy-dump -d test.hlto -p Test::Y >output
```

Comment:
```suggestion
# @TEST-EXEC: ${SCRIPTS}/printf '0' | spicy-dump -d test.hlto -p Test::Y >>output
```

Reply:
Good catch, I'll also update the baseline.